docs/src/main/sphinx/release/release-431.md

```diff
@@ -0,0 +1,55 @@
+# Release 431 (25 Oct 2023)
+
+## General
+
+* Add support for the `CREATE OR REPLACE TABLE` statement. ({issue}`13180`)
+* Disallow a offset of `null` in the {func}`lead` and {func}`lag` functions. ({issue}`19003`)
```

Comment:
`an`, but this sounds off to me.  Maybe:
```suggestion
* Disallow `null` value for the offset of the {func}`lead` and {func}`lag` functions. ({issue}`19003`)
```

Reply:
Disallow a `null` offset ...

Reply:
Adjusted